packages/cloud_firestore/cloud_firestore_platform_interface/lib/src/firestore_message_codec.dart

```diff
@@ -5,7 +5,11 @@
 part of cloud_firestore_platform_interface;
 
 @visibleForTesting
+// ignoring lint rule here as it's only visible for testing
+// ignore: public_member_api_docs
 class FirestoreMessageCodec extends StandardMessageCodec {
+  // ignoring lint rule here as it's only visible for testing
```

Comment:
Please add @visibleForTesting annotation as well, so it's skipped in the generated docs.